src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"></form>
+    <fieldset class="group">
+      <legend>Personal information</legend>
+      <label class="form-field">
+        Surname:
+        <input
+          type="text"
+          name="Surname"
+          required
+        >
+      </label>
+
+      <label class="form-field">
+        Name:
+        <input
+          type="text"
+          name="Name"
+          required
+        >
+      </label>
+      <label class="form-field">
+        How old are You?
+        <input
+          type="number"
+          name="Age"
+          min="1"
+          max="100"
+          placeholder="12"
+          required
+        >
+      </label>
+      <label class="form-field">
+        Full date of birth:
+        <input
+          type="data"
+          name="Data"
+          placeholder="dd. mm. yyyy"
+          required
+        >
+      </label>
+      <label class="form-field">
+        I accept the term of the agreement
+        <input
+          type="checkbox"
+          name="terms"
+          id="agreement"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="group">
+      <legend>Registration</legend>
+
+      <label class="form-field">
+        E-mail:
+        <input
+          type="email"
+          name="email"
+          required
+          placeholder="email@example.com"
+        >
+      </label>
+      <label class="form-field">
+        Password:
+        <input
+          type="password"
+          name="password"
+          required
+        >
+      </label>
+    </fieldset>
+    <fieldset class="group">
+      <legend>An interesting fact about you!</legend>
+      <label class="form-field">
+        Do you love cats?
+        <input
+          type="radio"
+          name="yes"
+        >
+        Yes
+        <input
+          type="radio"
+          name="no"
+        >
+        No
+      </label>
```

Comment:
only one variant could be chosen
solution: add to these 2 inputs the same name (loveCat), bit different values
check this link
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

<img width="276" alt="Screenshot 2022-09-07 at 10 53 29" src="https://user-images.githubusercontent.com/57598825/188821830-e3766f36-3b26-478d-8c79-f5eb0d47a83e.png">


```suggestion
        Do you love cats?
        <input
          type="radio"
          name="loveCats"
          value="yes"
        >
        Yes
        <input
          type="radio"
          name="loveCats"
          value="no"
        >
        No
      </label>
```